### PR TITLE
feat: add arbitrum uni token list

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of #4310 

Add uni's mapped arb1 token list

# To Test

1. Connect from US on arb1
* The new list should be used

### CODE REVIEW

File: libs/tokens/src/const/tokensLists.ts [hunk 1]
```diff
@@ -9,5 +9,6 @@ export const UNISWAP_TOKENS_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.
 export const GNOSIS_UNISWAP_TOKENS_LIST =
   'https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/GnosisUniswapTokensList.json'
 
-// TODO: add a uniswap mapped token list for arbitrum
-export const ARBITRUM_ONE_TOKENS_LIST = ''
+// TODO: update address once https://github.com/cowprotocol/token-lists/pull/393 is merged
+export const ARBITRUM_ONE_TOKENS_LIST =
+  'https://raw.githubusercontent.com/cowprotocol/token-lists/ca0fac81d0318234cf16b9f661d05a5c24c2ccb6/src/public/ArbitrumOneUniswapTokensList.json'
```
Comment (shoom3301) [inline libs/tokens/src/const/tokensLists.ts:14]: Should into `develop` / `main` in the future?

File: libs/tokens/src/state/tokenLists/tokenListsStateAtom.ts [hunk 2]
```diff
@@ -15,7 +20,13 @@ import {
 import { environmentAtom } from '../environmentAtom'
 
 const getUniswapTokenListUrl = (chainId: number) => {
-  return chainId === SupportedChainId.GNOSIS_CHAIN ? GNOSIS_UNISWAP_TOKENS_LIST : UNISWAP_TOKENS_LIST
+  if (chainId === SupportedChainId.GNOSIS_CHAIN) {
```
Comment (shoom3301) [inline libs/tokens/src/state/tokenLists/tokenListsStateAtom.ts:23]: `Record<SupportedChainId, string>`

File: libs/tokens/src/const/tokensLists.ts (hunk 1 above)
Comment (anxolin) [reply libs/tokens/src/const/tokensLists.ts:14]: I would think `main`

**Review by anxolin — APPROVED** [review]
Approving, but I agree with the suggestions from sasha

File: libs/tokens/src/const/tokensLists.ts (hunk 1 above)
Comment (alfetopito) [reply libs/tokens/src/const/tokensLists.ts:14]: It will! As soon as https://github.com/cowprotocol/token-lists/pull/393 is merged

**Review by shoom3301 — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**